src/Workspaces/Core/Portable/Shared/Utilities/SymbolEquivalenceComparer.GetHashCodeVisitor.cs

```diff
@@ -167,7 +167,22 @@ private int CombineHashCodes(IModuleSymbol x, int currentHash)
 
             private int CombineHashCodes(INamedTypeSymbol x, int currentHash)
             {
-                return Unwrap(x).Aggregate(currentHash, (a, n) => CombineNamedTypeHashCode(n, a));
+                currentHash = CombineNamedTypeHashCode(x, currentHash);
```

Comment:
are we expecting x to be actually same symbol when it is IErrorTypeSymbol? otherwise, does CandidateSymbols return symbols in same order between 2 different error type symbols? in another word, is CandidateSymbols has deterministic ordering or just random order?